package.json

```diff
@@ -1,14 +1,33 @@
 {
   "description": "Desktop application for Centar Srce",
   "license": "MIT",
+  "name": "srce_desktop_app",
+  "version": "1.0.0",
+  "homepage": "./",
   "repository": {
     "type": "git",
     "url": "https://github.com/nsoss/srce-desktop-app.git"
   },
   "scripts": {
     "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000 && electron .\"",
     "react-start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "pack": "electron-builder --dir",
+    "dist": "electron-builder",
+    "build-electron": "robocopy . build index.js ",
+    "build-src": "cd build && mkdir src && cd src && mkdir database && robocopy build/src/database src/database && cd .. && cd .. /S",
```

Comment:
Is there a way to write this without so many directory changes? It seems fragile—if the folder structure of the project is changed for whatever reason, this will break in non-obvious ways.

Reply:
I'm not sure how to find a file and copy it in one line.